docs/proposals/test-reports/Smart Coding benchmark suite Proposal_zh.md

```diff
@@ -0,0 +1,129 @@
+# 背景
```

Comment:
As mentioned, proposals shall be kept in the proposal PR. If it is needed in the test reports, then this document shall be written as test reports like [this link](https://ianvs.readthedocs.io/en/latest/proposals/test-reports/testing-single-task-learning-in-industrial-defect-detection-with-pcb-aoi.html).